src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/ThemeServiceTests.cs

```diff
@@ -1441,55 +1358,649 @@ await using (var context = InMemoryContentDbContext(contextId))
+            {
+                // Published with latest data
+                new()
+                {
+                    Release = new Release
+                    {
+                        Publication = publicationA,
+                        TimePeriodCoverage = TimeIdentifier.CalendarYear,
+                        ReleaseName = "2020",
+                        Type = ReleaseType.OfficialStatistics,
+                        Published = new DateTime(2020, 1, 1),
+                    },
+                    File = new File
+                    {
+                        Type = FileType.Data
+                    }
+                },
+                // Published without latest data
+                new()
+                {
+                    Release = new Release
+                    {
+                        Publication = publicationB,
+                        TimePeriodCoverage = TimeIdentifier.CalendarYear,
+                        ReleaseName = "2020",
+                        Type = ReleaseType.NationalStatistics,
+                        Published = new DateTime(2020, 1, 1),
+                    },
+                    File = new File
+                    {
+                        Type = FileType.Ancillary
+                    }
+                },
+                // Not published
+                new()
+                {
+                    Release = new Release
+                    {
+                        Publication = publicationC,
+                        TimePeriodCoverage = TimeIdentifier.CalendarYear,
+                        ReleaseName = "2020",
+                        Type = ReleaseType.AdHocStatistics,
+                    },
+                    File = new File
+                    {
+                        Type = FileType.Data
+                    }
+                },
+            };
+
+            var contextId = Guid.NewGuid().ToString();
+
+            await using (var context = InMemoryContentDbContext(contextId))
+            {
+                await context.AddAsync(theme);
+                await context.AddRangeAsync(releaseFiles);
+
+                await context.SaveChangesAsync();
+            }
+
+            await using (var context = InMemoryContentDbContext(contextId))
+            {
+                var service = BuildThemeService(context);
+
+                var result = await service.GetPublicationTree(PublicationTreeFilter.DataTables);
+
+                Assert.Single(result);
+                Assert.Equal("Theme A", result[0].Title);
+
+                Assert.Single(result[0].Topics);
+                Assert.Equal("Topic A", result[0].Topics[0].Title);
+
+                var publications = result[0].Topics[0].Publications;
+
+                Assert.Single(publications);
+                Assert.Equal("Publication A", publications[0].Title);
+            }
+        }
+
+        [Fact]
+        public async Task GetPublicationTree_DataTables_DoesNotFilterPublicationWithDataOnLatestRelease()
```

Comment:
Not sure this makes sense in the context of what the test actually asserts.  The latest release doesn't have any data on it, so it's actually filtered out.